paddle/cuda/include/hl_matrix.h

```diff
@@ -300,4 +300,12 @@ extern void hl_matrix_col2Vol(real* dataDst,
                               real alpha,
                               real beta);
 
+/**
+ * @brief  Matrix col2Vol: Convert col matrix into 3D volume
+ * @param[out]  out     output int vector.
+ * @param[in]   vec     input float vector.
+ * @param[in]   size    size of the vector.
+ */
+extern void hl_vector_cast2int(int* out, real* vec, int size);
+
```

Comment:
Maybe this functions is also needed add in the [file of `cuda/include/stub/hl_matrix_stub.h`]( https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/cuda/include/stub/hl_matrix_stub.h)  for the ONLY_CPU compiling.

Reply:
Oh, thanks, this should be the correct way instead of add `#ifdef` in `Vector.cpp`